app/konnect/org-management/teams-roles-users.md

```diff
@@ -0,0 +1,195 @@
+
+You must be part of the **Organization Admin** team to manage users, teams, and
+roles.
+
+## Manage teams and roles
+
+You can find a list of all teams in your organization through
+![](/assets/images/icons/konnect/icn-organizations.svg){:.inline .no-image-expand .konnect-icon}
+**Organization** > **Teams** in Konnect.
+
+* **Team:** A group of users with access to the same roles. Teams can be
+used to functionally separate departments or create hierarchical
+structure in an organization. Access to Konnect resources such as runtime groups
+and services can be shared with teams. A team can be granted granular
+access based on permissions and roles.
+
+* **Role:** Predefined access to a particular resource type (for example,
+  Services).
+
+When you create a Konnect account, you are automatically added to the Organization
+Admin team, which is one of the [predefined teams](/konnect/org-management/teams-reference)
+in Konnect. Predefined teams have sets of roles that can't be modified or
+deleted. You can add users to these teams, or create your own custom teams
+with any of the [supported roles](/konnect/org-management/roles-reference).
+
+Users can be part of any number of teams, and the roles gained from the teams
+are additive. For example, if you add a user to both the Service Developer and
+Portal Viewer teams, the user can create and manage Services
+through the Service Hub _and_ register applications through the Dev Portal.
+
+All users can view all teams and members of each team, but they can't view the
+team permissions.
+
+### Entity and role sharing
+
+An organization admin can share any role or entity with any user in the
+organization.
+
+Any user can also share Services or runtime groups that they have access to, with
+the same role or lesser.
+
+For example, say you have a Service Admin role:
+* You can share that Service with any other user through the Service Hub.
+* Since you have admin permissions, you can choose to share the Service any
+level of access: creator, deployer, viewer, etc.
+* The user that you share this Service with can then share the Service
+themselves.
+
+But what if you only have the Service Viewer role? In that case, you can only
+share the Service in read-only mode, as this is the lowest-level permission that
+you can have on the Service entity.
+
+You can [share any Service](/konnect/configure/servicehub/manage-services/#share-service)
+through the Service Hub, or
+[share any runtime group](/konnect/configure/runtime-manager/runtime-groups/manage/#share-runtime-group)
+through the Runtime Manager.
+
+### Create a new team
+
+1. From the left navigation menu in Konnect, open the
+![](/assets/images/icons/konnect/icn-organizations.svg){:.inline .no-image-expand .konnect-icon}
+ **Organization** link.
+2. Click **+ New Team**.
+3. Enter a team name and description. Both fields are required.
+4. Click **Save**.
+
+By default, your new team has the `Creator` role for all entities in the organization.
```

Comment:
This isn't a docs issue, but this isn't behaviour that I'd expect @haydenlam. Shouldn't we be defaulting to no permissions for a new team?

Reply:
@mheap You are correct. This was likely a bug in the Helium environment when @lena-larionova was writing this.

It has been fixed now. A team is created with no roles by default. Every single role must be intentionally assigned.